src/D8/MenuLinkContentTrait.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace IntegratedExperts\BehatSteps\D8;
+
+use Behat\Gherkin\Node\TableNode;
+use Drupal\menu_link_content\Entity\MenuLinkContent;
+
+/**
+ * Trait MenuLinkContentTrait.
+ *
+ * @package IntegratedExperts\BehatSteps\D8
+ */
+trait MenuLinkContentTrait {
+
+  /**
+   * Remove menu.
+   *
+   * Provide menu data in the following format:
+   *
+   * | id        |
+   * | 9991      |
+   * | ...       |
+   *
+   * @Given no menu_links:
```

Comment:
```
Given no "mymenu" menu links:
```

Reply:
Thanks this is a nice enhancement - I've adopted